ui/server.py

```diff
@@ -21,7 +21,7 @@
 app.config['UPLOAD_FOLDER'] = os.path.join(APP_ROOT, './backend')
 app.config['DEBUG'] = True  # Remove this line in production
 
-if os.getenv('USE_PG'):
+if os.getenv('USE_PG') is True:
```

Comment:
is `is True` needed?

Reply:
It is needed because `if os.getenv('USE_PG'):` only tests for the existence of the variable (if you have it set to `False` the condition is still true)